From: Shengjing Zhu <zhsj@debian.org>
Date: Tue, 6 Oct 2020 23:48:31 +0800
Subject: go1.14 cipher

Origin: backport, https://github.com/etcd-io/etcd/pull/11864
---
 pkg/tlsutil/cipher_suites.go | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/pkg/tlsutil/cipher_suites.go b/pkg/tlsutil/cipher_suites.go
index b5916bb..2150b6f 100644
--- a/pkg/tlsutil/cipher_suites.go
+++ b/pkg/tlsutil/cipher_suites.go
@@ -41,6 +41,9 @@ var cipherSuites = map[string]uint16{
 	"TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384": tls.TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,
 	"TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305":    tls.TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,
 	"TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305":  tls.TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,
+
+	"TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256":   tls.TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256,
+	"TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256": tls.TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256,
 }
 
 // GetCipherSuite returns the corresponding cipher suite,
